File: backend/models/driver.go
package models

type Driver struct {
    ID           int       `gorm:"primaryKey" json:"id"`
    DriverNumber int       `json:"driver_number"`
    FullName     string    `json:"full_name"`
    CountryCode  string    `json:"country_code"`
    TeamName     string    `json:"team_name"`
    TeamColour   string    `json:"team_colour"`
    PictureURL   string    `json:"picture_url"`
    Sessions     []Session `gorm:"many2many:driver_sessions;" json:"-"`
}

File: frontend/src/types/driver.ts
export interface Driver {
    driverNumber: number;
    fullName: string;
    countryCode: string;
    teamName: string;
    teamColour: string;
    pictureURL: string;
}

export interface DriverResponse {
    id: number;
    driver_number: number;
    full_name: string;
    country_code: string;
    team_name: string;
    team_colour: string;
    picture_url: string;
}
